src/pages/workspaces/workspace/Analyses.js

```diff
@@ -228,7 +229,8 @@ const Analyses = _.flow(
   analysesData: { apps, refreshApps, runtimes, refreshRuntimes, appDataDisks, persistentDisks },
   onRequesterPaysError
 }, _ref) => {
-  // State
+  // TODO: revert after testing
+  // var googleProject = false
```

Comment:
This looks like a type mismatch. Would undefined or null work here?
Also, `let` gives you mutability with the scope-sanity of `const`.
```suggestion
  // let googleProject = undefined
```

Reply:
Curious what the use case requiring a mutable variable for `googleProject` would be

Reply:
This was just a temporary change to mock having an Azure workspace (see the TODO).